proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -24,40 +24,34 @@ However, with this proposal, by adding a new parameter, the SDL app always estab
-    ii. Core will check its current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
-    iii. If there is no Audio support, Core will check if it supports the auto connect BT function. If it does it will continue in flow.
-    iv. If `requiresAudioSupport` is true and there is no supported audio methods and Core does not support the auto connect BT function, it will send a `StartServiceNAK` with the reason `No audio support available`.
-4. If it supports the auto connect BT function, a new param in the payload, `autoBTCapability` is set to true, otherwise set to false. Then if Core has continued, it will send a `StartServiceACK` with new param `autoBTCapability` set to true.
+3. Core will receive the `StartService` for the RPC service: 
+    i. Core will check it's current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
+    ii. If there is no Audio support, core will check if it supports the auto connect BT feature. If it does it will continue in flow.
+    iii. If there is no supported audio methods and Core does not support the auto connect BT feature, it will send a `StartServiceNAK` with the reason `No audio support available`.
+4. If Core has continued, it will send a `StartServiceACK` with a new param `autoBTCapability` set to true.
 5. The app receives the response to its `StartService` for the RPC service:
-    i. If the response was a `StartServiceNAK`, the app will shut down.
-    ii. If the response was a `StartServiceACK`, it will continue in flow.
-    iii. If `requiresAudioSupport` was set to false or not set, it will continue in flow.
-    iv. The library check it's current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
-    v. If `autoBTCapability` was set to true, t will continue in flow.
-    vi. If the response was a `StartServiceACK` and `requiresAudioSupport` was set to true and autoBTCapability was set to false, the app will shutdown.
-6. The app will send its RegisterAppInterface which will include `requiresAudioSupport` and `bluetoothDeviceAddress` in `deviceInfo`.
-7. When core receives RegisterAppInterface, it sends response. Then it will send `OnAppRegistered` include `deviceInfo` to HMI.
+    1.If the response was a `StartServiceNAK` the app will shutdown.
+    2. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, but the protocol version of the ACK is less than the major version of this feature, the app will shutdown.
+    3. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, the protocol version of the ACK is equal or greater than the major version of this feature, and the `autoBTCapability` flag is set to false, the app will check if audio support is available using the `MediaStreamingStatus` class. If it is available it will continue, if not it will shutdown.
```

Comment:
```suggestion
    3. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, the protocol version of the ACK is equal or greater than the major version of this feature, and the `autoBTCapability` flag is set to false, the app will check if audio support is available using the `MediaStreamingStatus` class. If it is available it will continue, if not it will shut down.
```